invenio_previewer/extensions/ipynb.py

```diff
@@ -28,7 +28,7 @@ def render(file):
     notebook = nbformat.reads(content.decode("utf-8"), as_version=4)
 
     html_exporter = HTMLExporter()
-    html_exporter.template_file = "basic"
+    html_exporter.template_file = "base"
```

Comment:
In the docs, they seem to be specifying the `classic` and `lab` templates as well, which generate also inline CSS and use a CDN for loading some JS... I see also [this comment](https://github.com/jupyter/nbconvert/pull/1431#issuecomment-725512612), wondering if that's the expected behaviour.

We can shelve this as a separate and explore if other templates might be better for our use-case.

Reply:
Regarding bump: `base` appeared as a renaming of `basic` [here](https://github.com/jupyter/nbconvert/commit/f4c7daefe0e24055a92f185d6fd803abf909db2f) and this was released on `6.0.0a`. 

Regarding other templates: I agree that we should investigate it as an improvement, I created a [card](https://github.com/orgs/inveniosoftware/projects/97#card-83365296) for that.